scripts/monitoring/cron-send-docker-timer-hawkular.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env python2
+# vim: expandtab:tabstop=4:shiftwidth=4
+
+'''
+    Report time to execute 'docker info'
+'''
+
+# Adding the ignore because it does not like the naming of the script
+# to be different than the class name
+# pylint: disable=invalid-name
+
+
+from docker import AutoVersionClient
+from docker.errors import DockerException
+#from openshift_tools.monitoring.zagg_sender import ZaggSender
+from openshift_tools.monitoring.hawk_sender import HawkSender as ZaggSender
+from openshift_tools.timeout import TimeoutException
+from openshift_tools.monitoring.dockerutil import DockerUtil
+import json
+import time
+
+if __name__ == "__main__":
+    keys = None
+    zs = ZaggSender()
```

Comment:
I import HawkSender as ZaggSender
`from openshift_tools.monitoring.hawk_sender import HawkSender as ZaggSender`

I did not want to change the rest of the file ...